src/pyhf/infer/mle.py

```diff
@@ -0,0 +1,65 @@
+"""Module for Maximum Likelihood Estimation."""
+from .. import get_backend
+
+
+def twice_nll(pars, data, pdf):
```

Comment:
`nll` is a sliiiiiiightly bothersome name for HEP physicists (NLO, NLL, etc...). Is there no official name for this calculable?

Reply:
NLL (see e.g. createNLL in RooFit) is fairly standard. If it were a likelihood ratio it'd be https://en.wikipedia.org/wiki/Deviance_(statistics) but it's not